### PR TITLE
chore: sort instance types and infos return data

### PR DESCRIPTION
**Pull request by Vacant2333**
Fixes #

## Proposed Changes

* chore: sort instance types and infos return data

**Release Note**

```release-note

```

### CODE REVIEW

**Review by jwcesign — APPROVED** [review]
/lgtm

File: pkg/client/aws.go [hunk 1]
```diff
@@ -823,7 +866,7 @@ func (a *AWSPriceClient) GetInstancePrice(region, instanceType string) *apis.Ins
 	return d
 }
 
-func (a *AWSPriceClient) ListInstanceTypes() []string {
+func (a *AWSPriceClient) ListInstanceTypes() []*Instance {
```
Comment (jwcesign) [inline pkg/client/aws.go:869]: Does this change the API response?

Comment (Vacant2333) [reply pkg/client/aws.go:869]: yes, and its a new api for now, so its a acceptable(and better) change for the frontend

Comment (Vacant2333) [reply pkg/client/aws.go:869]: https://github.com/cloudpilot-ai/priceserver/pull/12/files#diff-a8603ad7273898c7bf99cbffa08193562c9bb15d0de146abbf8b240aa25054f0R826

**Pull request merged**